client/components/main/About.jsx

```diff
@@ -4,10 +4,9 @@ import {
   Container,
   Grid,
 } from '@material-ui/core';
-import Image from 'material-ui-image';
-import partner1 from '@assets/Partners_icon_1.png';
-import partner2 from '@assets/Partners_icon_2.png';
-import partner3 from '@assets/Partners_icon_3.png';
+import partner1 from '@assets/empower_la_logo.png';
```

Comment:
nit: let's give the imports more meaningful names as well, e.g.,:
`import empower_la_logo from '@assets/empower_la_logo.png';`